src/utils/filterPeople.ts

```diff
@@ -0,0 +1,18 @@
+import { Person } from '../types/Person';
+
+export function filterPeople(
+  people: Person[],
+  { appliedQuery }: { appliedQuery: string },
+) {
+  let filteredPeople = [...people];
+
+  if (appliedQuery) {
+    const normalizedQuery = appliedQuery.toLocaleLowerCase().trim();
```

Comment:
The `toLocaleLowerCase()` method is a good approach to make the search case-insensitive. However, please note that for some languages and locales, this method might not work as expected. If you plan to support multiple languages, you might need to use a library for more accurate case conversion.